### DISCUSSION TITLE
Is there a way to pass payload along with the url for the dynamic badges?

### DISCUSSION THREAD

**Discussion by ThishaniLucas** [badges/shields#9667] [Q&A] [answered]
$subject?
Basiaclly, along with https://img.shields.io/badge/dynamic/json?url=http%3Atest.com&query=%24, I need to pass a payload as well to the url.

**Comment by chris48s** [comment]
Can you explain what you mean by the word "payload" here, or give an example of what you are trying to achieve?

**Reply by ThishaniLucas** [reply]
The URL I'm trying to access requires an HTTP POST. So I need to call the URL with a JSON payload. In the currently-supported 'url' query param, I can pass only the URL I'm trying to access.

**Reply by chris48s** [reply, marked as answer]
I see. No - the dynamic badges can only make a GET request.
